internal/gatewayapi/translator_test.go

```diff
@@ -94,6 +95,21 @@ func TestTranslate(t *testing.T) {
 				},
 			)
 
+			resources.ServiceImports = append(resources.ServiceImports,
```

Comment:
prefer if we dont add this by default here, but in the YAML testdata input, since this resource is not the common case